File: app/src/main/java/woo/sopt22/meowbox/View/MyPage/ProgressBar/OnStateItemClickListener.java
package woo.sopt22.meowbox.View.MyPage.ProgressBar;

import woo.sopt22.meowbox.View.MyPage.ProgressBar.StateProgressBar;
import woo.sopt22.meowbox.View.MyPage.ProgressBar.StateItem;

public interface OnStateItemClickListener {

    void onStateItemClick(StateProgressBar stateProgressBar, StateItem stateItem, int stateNumber, boolean isCurrentState);

}

File: app/src/main/java/woo/sopt22/meowbox/View/MyPage/ProgressBar/OnStateItemClickListener.kt
package woo.sopt22.meowbox.View.MyPage.ProgressBar

import woo.sopt22.meowbox.View.MyPage.ProgressBar.StateProgressBar
import woo.sopt22.meowbox.View.MyPage.ProgressBar.StateItem

interface OnStateItemClickListener {

    fun onStateItemClick(stateProgressBar: StateProgressBar, stateItem: StateItem, stateNumber: Int, isCurrentState: Boolean)

}